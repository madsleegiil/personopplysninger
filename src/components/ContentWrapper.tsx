import * as React from 'react';

const ContentWrapper = () => {
    return (
        <div className="wrapper">
            Jepp
        </div>
    );
};

export default ContentWrapper;
