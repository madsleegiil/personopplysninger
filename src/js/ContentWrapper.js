import React, { Component } from 'react';
import PropTypes from 'prop-types';
import Personalia from './components/Personalia';
import AdresseContainer from './containers/AdresseContainer';
import LinksContainer from './containers/LinksContainer';
import AlternativListe from './components/AlternativListe';
import FooterInfo from './components/FooterInfo';

class ContentWrapper extends Component {
  render() {
    return (
      <div>
        <Personalia
          navn={this.props.userInfo.navn}
          ident={this.props.userInfo.ident}
          statsborgerskap={this.props.userInfo.statsborgerskap}
          status={this.props.userInfo.status}
          telefon={this.props.userInfo.telefon}
          tiltak={this.props.userInfo.tiltak}
          kjonn={this.props.userInfo.kjonn}
          spraak={this.props.userInfo.spraak}
          sivilstand={this.props.userInfo.sivilstand}
          kontonummer={this.props.userInfo.kontonummer}
          spesreg={this.props.userInfo.spesreg}
        />
        <AdresseContainer
          adresseInfo={this.props.userInfo.adresseinfo}
        />
        <LinksContainer />
        <AlternativListe />
        <FooterInfo />
      </div>
    );
  }
}

const datoKildeTypeShape = PropTypes.shape({
  datoFraOgMed: PropTypes.string,
  kilde: PropTypes.string,
  type: PropTypes.string,
});

const sivilstandShape = PropTypes.shape({
  datoFraOgMed: PropTypes.string,
  kilde: PropTypes.string,
  kode: PropTypes.shape({
    kodeverk: PropTypes.string,
    verdi: PropTypes.string,
  }),
});

ContentWrapper.propTypes = {
  userInfo: PropTypes.shape({
    adresseinfo: PropTypes.shape({
      boadresse: PropTypes.shape({
        adresse: PropTypes.string,
        adressetillegg: PropTypes.string,
        bydel: PropTypes.string,
        datoFraOgMed: PropTypes.string,
        kilde: PropTypes.string,
        kommune: PropTypes.string,
        landkode: PropTypes.string,
        matrikkeladresse: PropTypes.shape({
          bruksnummer: PropTypes.string,
          festenummer: PropTypes.string,
          gaardsnummer: PropTypes.string,
          undernummer: PropTypes.string,
        }),
        postnummer: PropTypes.string,
        veiadresse: PropTypes.shape({
          bokstav: PropTypes.string,
          bolignummer: PropTypes.string,
          gatekode: PropTypes.string,
          husnummer: PropTypes.string,
        }),
      }),
      geografiskTilknytning: PropTypes.shape({
        bydel: PropTypes.string,
        datoFraOgMed: PropTypes.string,
        kilde: PropTypes.string,
        kommune: PropTypes.string,
        land: PropTypes.string,
      }),
      postadresse: PropTypes.string,
      prioritertAdresse: PropTypes.string,
      tilleggsadresse: PropTypes.string,
      utenlandskAdresse: PropTypes.string,
    }),
    barnListe: PropTypes.arrayOf(PropTypes.shape({
      datoRegistert: PropTypes.string,
      egenansatt: PropTypes.bool,
      fnr: PropTypes.string,
      kilde: PropTypes.string,
      spesreg: PropTypes.string,
    })),
    brukerbehovListe: PropTypes.arrayOf(PropTypes.shape({
      behov: PropTypes.string,
      datoFraOgMed: PropTypes.string,
      kilde: PropTypes.string,
    })),
    datoFraOgMed: PropTypes.string,
    doedsdato: PropTypes.shape({
      dato: PropTypes.string,
      datoFraOgMed: PropTypes.string,
      kilde: PropTypes.string,
    }),
    egenansatt: PropTypes.shape({
      datoFraOgMed: PropTypes.string,
      erEgenansatt: PropTypes.bool,
      kilde: PropTypes.string,
    }),
    foedselsdato: PropTypes.string,
    foedtIKommune: PropTypes.string,
    foedtILand: PropTypes.string,
    foreldreansvar: datoKildeTypeShape,
    ident: PropTypes.string,
    identtype: PropTypes.shape({
      kodeverk: PropTypes.string,
      verdi: PropTypes.string,
    }),
    innvandringUtvandring: PropTypes.shape({
      innvandretDato: PropTypes.string,
      innvandretKilde: PropTypes.string,
      innvandretLand: PropTypes.shape({
        kodeverk: PropTypes.string,
        verdi: PropTypes.string,
      }),
      utvandretDato: PropTypes.string,
      utvandretKilde: PropTypes.string,
      utvandretLand: PropTypes.shape({
        kodeverk: PropTypes.string,
        verdi: PropTypes.string,
      }),
    }),
    kilde: PropTypes.string,
    kjonn: PropTypes.string,
    kontonummer: PropTypes.shape({
      datoFraOgMed: PropTypes.string,
      kilde: PropTypes.string,
      nummer: PropTypes.string,
    }),
    navn: PropTypes.shape({
      datoFraOgMed: PropTypes.string,
      forkortetNavn: PropTypes.string,
      fornavn: PropTypes.string,
      kilde: PropTypes.string,
      mellomnavn: PropTypes.string,
      slektsnavn: PropTypes.string,
      slektsnavnUgift: PropTypes.string,
    }),
    oppholdstillatelse: datoKildeTypeShape,
    relasjon: PropTypes.shape({
      datoFraOgMed: PropTypes.string,
      egenansatt: PropTypes.bool,
      fnr: PropTypes.string,
      kilde: PropTypes.string,
      spesreg: PropTypes.string,
      type: PropTypes.string,
    }),
    samboer: PropTypes.shape({
      datoFraOgMed: PropTypes.string,
      fnr: PropTypes.string,
      kilde: PropTypes.string,
    }),
    sivilstand: sivilstandShape,
    spesreg: sivilstandShape,
    spraak: sivilstandShape,
    statsborgerskap: sivilstandShape,
    status: sivilstandShape,
    telefon: PropTypes.shape({
      jobb: PropTypes.string,
      jobbDatoRegistrert: PropTypes.string,
      jobbKilde: PropTypes.string,
      mobil: PropTypes.string,
      mobilDatoRegistrert: PropTypes.string,
      mobilKilde: PropTypes.string,
      privat: PropTypes.string,
      privatDatoRegistrert: PropTypes.string,
      privatKilde: PropTypes.string,
    }),
    tiltak: PropTypes.shape({
      datoFraOgMed: PropTypes.string,
      datoTil: PropTypes.string,
      kilde: PropTypes.string,
      type: PropTypes.string,
    }),
    utenlandsinfoList: PropTypes.arrayOf(PropTypes.shape({
      datoFraOgMed: PropTypes.string,
      familienavnFodt: PropTypes.string,
      farsFamilenavn: PropTypes.string,
      farsFornavn: PropTypes.string,
      foedested: PropTypes.string,
      fornavnFodt: PropTypes.string,
      idOff: PropTypes.string,
      institusjon: PropTypes.string,
      institusjonNavn: PropTypes.string,
      kilde: PropTypes.string,
      kildePin: PropTypes.string,
      land: PropTypes.string,
      morsFamilenavn: PropTypes.string,
      morsFornavn: PropTypes.string,
      nasjonalId: PropTypes.string,
      nasjonalitet: PropTypes.string,
      sedRef: PropTypes.string,
      sektor: PropTypes.string,
    })),
    utenlandskBank: PropTypes.shape({
      adresse1: PropTypes.string,
      adresse2: PropTypes.string,
      adresse3: PropTypes.string,
      bankkode: PropTypes.string,
      banknavn: PropTypes.string,
      datoFraOgMed: PropTypes.string,
      iban: PropTypes.string,
      kilde: PropTypes.string,
      kontonummer: PropTypes.string,
      land: PropTypes.string,
      swiftkode: PropTypes.string,
      valuta: PropTypes.string,
    }),
    vergemaalListe: PropTypes.arrayOf(PropTypes.shape({
      aaksId: PropTypes.string,
      datoFraOgMed: PropTypes.string,
      egenansatt: PropTypes.bool,
      embete: PropTypes.string,
      fnr: PropTypes.string,
      id: PropTypes.string,
      kilde: PropTypes.string,
      mandattype: PropTypes.string,
      sakstype: PropTypes.string,
      spesreg: PropTypes.string,
      type: PropTypes.string,
      vedtaksdato: PropTypes.string,
    })),
  }).isRequired,
};

export default ContentWrapper;
