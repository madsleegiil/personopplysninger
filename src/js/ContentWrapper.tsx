import React from "react";
import Header from "./components/Header";
import Personalia from "./components/Personalia";
import AdresseContainer from "./containers/AdresseContainer";
import LinksContainer from "./containers/LinksContainer";
import AlternativListe from "./components/AlternativListe";
import { formatName } from "./utils/text";

const ContentWrapper = ({ personalia, adresser }: any) => (
  <div className="Content">
    <Header fornavn={formatName(personalia.fornavn)} />
    <Personalia
      fornavn={formatName(personalia.fornavn)}
      etternavn={formatName(personalia.etternavn)}
      personident={personalia.personident}
      kontonr={personalia.kontonr}
      tlfnr={personalia.tlfnr}
      spraak={personalia.spraak}
      epostadr={personalia.epostadr}
      statsborgerskap={personalia.statsborgerskap}
      foedested={personalia.foedested}
      sivilstand={personalia.sivilstand}
      kjoenn={personalia.kjoenn}
    />
    <AdresseContainer adresseInfo={adresser} />
    <LinksContainer />
    <AlternativListe />
  </div>
);

export default ContentWrapper;
