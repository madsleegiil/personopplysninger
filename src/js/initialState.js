export default {
  adresseinfo: {
    boadresse: {
      adresse: '',
      adressetillegg: '',
      bydel: '',
      datoFraOgMed: '',
      kilde: '',
      kommune: '',
      land: '',
      matrikkeladresse: {
        bruksnummer: '',
        festenummer: '',
        gaardsnummer: '',
        undernummer: '',
      },
      postnummer: '',
      poststed: '',
      veiadresse: {
        bokstav: '',
        bolignummer: '',
        gatekode: '',
        husnummer: '',
      },
    },
    geografiskTilknytning: {
      bydel: '',
      datoFraOgMed: '',
      kilde: '',
      kommune: '',
      land: '',
    },
    postadresse: {
      adresse1: '',
      adresse2: '',
      adresse3: '',
      datoFraOgMed: '',
      kilde: '',
      land: '',
      postnummer: '',
    },
    prioritertAdresse: {
      datoFraOgMed: '',
      kilde: '',
      type: '',
    },
    tilleggsadresse: {
      adresse1: '',
      adresse2: '',
      adresse3: '',
      bolignummer: '',
      bydel: '',
      datoFraOgMed: '',
      gateKode: '',
      husbokstav: '',
      husnummer: '',
      kilde: '',
      kommunenummer: '',
      postboksanlegg: '',
      postboksnummer: '',
      postnummer: '',
    },
    utenlandskAdresse: {
      adresse1: '',
      adresse2: '',
      adresse3: '',
      datoFraOgMed: '',
      kilde: '',
    },
  },
  barnListe: [
    {
      datoRegistert: '',
      egenansatt: null,
      fnr: '',
      kilde: '',
      spesreg: '',
    },
  ],
  brukerbehovListe: [
    {
      behov: '',
      datoFraOgMed: '',
      kilde: '',
    },
  ],
  datoFraOgMed: '',
  doedsdato: {
    dato: '',
    datoFraOgMed: '',
    kilde: '',
  },
  egenansatt: {
    datoFraOgMed: '',
    erEgenansatt: null,
    kilde: '',
  },
  foedselsdato: '',
  foedtIKommune: '',
  foedtILand: '',
  foreldreansvar: {
    datoFraOgMed: '',
    kilde: '',
    type: '',
  },
  ident: '',
  identtype: '',
  innvandringUtvandring: {
    innvandretDato: '',
    innvandretKilde: '',
    innvandretLand: '',
    utvandretDato: '',
    utvandretKilde: '',
    utvandretLand: '',
  },
  kilde: '',
  kjonn: '',
  kontonummer: {
    datoFraOgMed: '',
    kilde: '',
    nummer: '',
  },
  navn: {
    datoFraOgMed: '',
    forkortetNavn: '',
    fornavn: '',
    kilde: '',
    mellomnavn: '',
    slektsnavn: '',
    slektsnavnUgift: '',
  },
  oppholdstillatelse: {
    datoFraOgMed: '',
    kilde: '',
    type: '',
  },
  relasjon: {
    datoFraOgMed: '',
    egenansatt: null,
    fnr: '',
    kilde: '',
    spesreg: '',
    type: '',
  },
  samboer: {
    datoFraOgMed: '',
    fnr: '',
    kilde: '',
  },
  sivilstand: {
    datoFraOgMed: '',
    kilde: '',
    kode: '',
  },
  spesreg: {
    datoFraOgMed: '',
    kilde: '',
    kode: '',
  },
  spraak: {
    datoFraOgMed: '',
    kilde: '',
    kode: '',
  },
  statsborgerskap: {
    datoFraOgMed: '',
    kilde: '',
    kode: '',
  },
  status: {
    datoFraOgMed: '',
    kilde: '',
    kode: '',
  },
  telefon: {
    jobb: '',
    jobbDatoRegistrert: '',
    jobbKilde: '',
    mobil: '',
    mobilDatoRegistrert: '',
    mobilKilde: '',
    privat: '',
    privatDatoRegistrert: '',
    privatKilde: '',
  },
  tiltak: {
    datoFraOgMed: '',
    datoTil: '',
    kilde: '',
    type: '',
  },
  utenlandsinfoList: [
    {
      datoFraOgMed: '',
      familienavnFodt: '',
      farsFamilenavn: '',
      farsFornavn: '',
      foedested: '',
      fornavnFodt: '',
      idOff: '',
      institusjon: '',
      institusjonNavn: '',
      kilde: '',
      kildePin: '',
      land: '',
      morsFamilenavn: '',
      morsFornavn: '',
      nasjonalId: '',
      nasjonalitet: '',
      sedRef: '',
      sektor: '',
    },
  ],
  utenlandskBank: {
    adresse1: '',
    adresse2: '',
    adresse3: '',
    bankkode: '',
    banknavn: '',
    datoFraOgMed: '',
    iban: '',
    kilde: '',
    kontonummer: '',
    land: '',
    swiftkode: '',
    valuta: '',
  },
  vergemaalListe: [
    {
      aaksId: '',
      datoFraOgMed: '',
      egenansatt: null,
      embete: '',
      fnr: '',
      id: '',
      kilde: '',
      mandattype: '',
      sakstype: '',
      spesreg: '',
      type: '',
      vedtaksdato: '',
    },
  ],
};
