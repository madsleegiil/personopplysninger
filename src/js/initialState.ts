export default {
  loading: true,
  personalia: {
    fornavn: "",
    etternavn: "",
    fnr: "",
    kontonr: "",
    tlfnr: null,
    spraak: null,
    epostadr: "",
    personstatus: "",
    statsborgerskap: "",
    foedested: "",
    sivilstand: "",
    kjoenn: "",
    datakilder: [{}]
  },
  adresser: {
    boadresse: {
      adresse: "",
      adressetillegg: "",
      bydel: "",
      datoFraOgMed: "",
      kommune: "",
      land: "",
      matrikkeladresse: {
        bruksnummer: null,
        festenummer: null,
        gaardsnummer: null,
        undernummer: null
      },
      postnummer: "",
      poststed: "",
      veiadresse: {
        bokstav: "",
        bolignummer: "",
        gatekode: "",
        husnummer: ""
      }
    },
    geografiskTilknytning: {
      bydel: "",
      datoFraOgMed: "",
      kommune: "",
      land: ""
    },
    postadresse: null,
    prioritertAdresse: null,
    tilleggsadresse: null,
    utenlandskAdresse: null,
    datakilder: [{}]
  }
};
