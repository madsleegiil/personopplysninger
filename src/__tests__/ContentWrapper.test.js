import * as React from 'react';
import ContentWrapper from 'js/ContentWrapper';
import wrapIntl from 'js/IntlTestHelper';
import initialState from '../js/initialStateNy';
import ShallowRenderer from 'react-test-renderer/shallow';

test('render ContentWrapper with content', () => {
  const renderer = new ShallowRenderer()
  const result = renderer.render(wrapIntl(<ContentWrapper userInfo={initialState} />));
  expect(result).toMatchSnapshot()
});
