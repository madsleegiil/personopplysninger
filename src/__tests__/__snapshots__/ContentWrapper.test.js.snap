// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render ContentWrapper with content 1`] = `
<div>
  <div
    className="BoxContainer box-bottom-margin"
  >
    <div
      className="BoxWithHeader"
    >
      <h1>
        Personalia
      </h1>
      <div
        className="icon-box-wrapper"
      >
        <img
          alt=""
          className="box-icon"
          src="woman.png"
        />
        <div
          className="Box"
        >
          <ul
            className="personalia-list"
          >
            <li>
              <span
                className="title"
              >
                <span>
                  Fornavn
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Etternavn
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Fødselsnummer
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Utenlandsk ID
                </span>
              </span>
              <span
                className="content"
              >
                x
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Telefon
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Kontonummer
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  e-post
                </span>
              </span>
              <span
                className="content"
              >
                x
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Språk
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Statsborgerskap
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Personstatus
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Fødested
                </span>
              </span>
              <span
                className="content"
              >
                x
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Spesialregistrering
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Sivilstand
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Sikkerhetstiltak
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Kjønn
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Tilpasset kommunikasjon
                </span>
              </span>
              <span
                className="content"
              >
                x
              </span>
            </li>
          </ul>
          <div
            className="box-footer"
          >
            <span>
              Kilde: NAV, Skatteetaten, Folkeregisteret, Kontakt- og reservasjonsregisteret
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div>
    <div
      className="BoxContainer box-bottom-margin-small"
    >
      <div
        className="BoxWithHeader"
      >
        <h1>
          Adresse
        </h1>
        <div
          className="icon-box-wrapper"
        >
          <img
            alt=""
            className="box-icon"
            src="house.png"
          />
          <div
            className="Box"
          >
            <div
              className="address-box"
            >
              <h3
                className="address-type"
              >
                Bostedsadresse
              </h3>
              <ul
                className="address-list-col-4"
              >
                <li
                  className="address-col"
                >
                  <span
                    className="title"
                  >
                    <span>
                      Gate
                    </span>
                  </span>
                  <span
                    className="content"
                  >
                    
                  </span>
                </li>
                <li
                  className="address-col"
                >
                  <span
                    className="title"
                  >
                    <span>
                      Postnummer
                    </span>
                  </span>
                  <span
                    className="content"
                  >
                    
                  </span>
                </li>
                <li
                  className="address-col"
                >
                  <span
                    className="title"
                  >
                    <span>
                      Poststed
                    </span>
                  </span>
                  <span
                    className="content"
                  >
                    Kristiansand
                  </span>
                </li>
              </ul>
              <div
                className="box-footer"
              >
                Kilde: 
                
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="BoxContainer box-bottom-margin-small"
    >
      <div
        className="BoxWithHeader"
      >
        <div
          className="icon-box-wrapper"
        >
          <div
            className="box-icon"
          />
          <div
            className="Box"
          >
            <div
              className="address-box"
            >
              <h3
                className="address-type"
              >
                Postadresse
              </h3>
              <ul
                className="address-list-col-3"
              >
                <li
                  className="address-col"
                >
                  <span
                    className="title"
                  >
                    <span>
                      Adresselinje 1
                    </span>
                  </span>
                  <span
                    className="content"
                  >
                    
                  </span>
                </li>
                <li
                  className="address-col"
                >
                  <span
                    className="title"
                  >
                    <span>
                      Adresselinje 2
                    </span>
                  </span>
                  <span
                    className="content"
                  >
                    
                  </span>
                </li>
                <li
                  className="address-col"
                >
                  <span
                    className="title"
                  >
                    <span>
                      Adresselinje 3
                    </span>
                  </span>
                  <span
                    className="content"
                  >
                    
                  </span>
                </li>
                <li
                  className="address-col"
                >
                  <span
                    className="title"
                  >
                    <span>
                      Postnummer
                    </span>
                  </span>
                  <span
                    className="content"
                  >
                    
                  </span>
                </li>
                <li
                  className="address-col"
                >
                  <span
                    className="title"
                  >
                    <span>
                      Land
                    </span>
                  </span>
                  <span
                    className="content"
                  >
                    
                  </span>
                </li>
              </ul>
              <div
                className="box-footer"
              >
                Kilde: 
                
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="BoxContainer box-bottom-margin"
    >
      <div
        className="BoxWithHeader"
      >
        <div
          className="icon-box-wrapper"
        >
          <div
            className="box-icon"
          />
          <div
            className="Box"
          >
            <div
              className="address-box"
            >
              <h3
                className="address-type"
              >
                Utenlandsk adresse
              </h3>
              <ul
                className="address-list-col-3"
              >
                <li
                  className="address-col"
                >
                  <span
                    className="title"
                  >
                    <span>
                      Adresselinje 1
                    </span>
                  </span>
                  <span
                    className="content"
                  >
                    
                  </span>
                </li>
                <li
                  className="address-col"
                >
                  <span
                    className="title"
                  >
                    <span>
                      Adresselinje 2
                    </span>
                  </span>
                  <span
                    className="content"
                  >
                    
                  </span>
                </li>
                <li
                  className="address-col"
                >
                  <span
                    className="title"
                  >
                    <span>
                      Adresselinje 3
                    </span>
                  </span>
                  <span
                    className="content"
                  >
                    
                  </span>
                </li>
              </ul>
              <div
                className="box-footer"
              >
                Kilde: 
                
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="BoxContainer box-bottom-margin"
  >
    <div
      className="BoxWithHeader"
    >
      <h1>
        Familierelasjoner
      </h1>
      <div
        className="icon-box-wrapper"
      >
        <div
          className="box-icon"
        />
        <div
          className="Box"
        >
          <h3>
            Familierelasjoner
          </h3>
        </div>
      </div>
    </div>
  </div>
  <div
    className="BoxContainer box-bottom-margin"
  >
    <div
      className="BoxWithHeader"
    >
      <h1>
        Arbeidsforhold
      </h1>
      <div
        className="icon-box-wrapper"
      >
        <div
          className="box-icon"
        />
        <div
          className="Box"
        >
          <h3>
            Arbeidsforhold
          </h3>
        </div>
      </div>
    </div>
  </div>
  <div
    className="BoxContainer box-bottom-margin"
  >
    <div
      className="BoxWithHeader"
    >
      <h1>
        Pensjon
      </h1>
      <div
        className="icon-box-wrapper"
      >
        <div
          className="box-icon"
        />
        <div
          className="Box"
        >
          <div
            className="link-box-content"
          >
            <div
              className="link-info"
            >
              NAV har opplysninger om din pensjon i en egen tjeneste.
            </div>
            <a
              href="https://www.nav.no/no/Person/Pensjon/Hva+kan+jeg+fa+i+pensjon"
            >
              Gå til Din pensjon
            </a>
            <div
              className="box-footer link-footer"
            >
              Kilde: 
              Pensjonsregisteret
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="BoxContainer box-bottom-margin"
  >
    <div
      className="BoxWithHeader"
    >
      <h1>
        Dine saker
      </h1>
      <div
        className="icon-box-wrapper"
      >
        <div
          className="box-icon"
        />
        <div
          className="Box"
        >
          <div
            className="link-box-content"
          >
            <div
              className="link-info"
            >
              Oversikt over dine saker og kommunikasjon med NAV
            </div>
            <a
              href="https://tjenester.nav.no/saksoversikt/app"
            >
              Gå til saksoversikt
            </a>
            <div
              className="box-footer link-footer"
            >
              Kilde: 
              NAV
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="BoxContainer box-bottom-margin"
  >
    <div
      className="BoxWithHeader"
    >
      <h1>
        Utbetalinger
      </h1>
      <div
        className="icon-box-wrapper"
      >
        <div
          className="box-icon"
        />
        <div
          className="Box"
        >
          <div
            className="link-box-content"
          >
            <div
              className="link-info"
            >
              Oversikt over dine utbetalinger fra NAV
            </div>
            <a
              href="https://www.nav.no/no/NAV+og+samfunn/Kontakt+NAV/Utbetalinger/Utbetalinger"
            >
              Gå til utbetalinger
            </a>
            <div
              className="box-footer link-footer"
            >
              Kilde: 
              NAV
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="BoxContainer box-bottom-margin"
  >
    <div
      className="BoxWithHeader"
    >
      <h1>
        Jobbsøk
      </h1>
      <div
        className="icon-box-wrapper"
      >
        <div
          className="box-icon"
        />
        <div
          className="Box"
        >
          <div
            className="link-box-content"
          >
            <div
              className="link-info"
            >
              Jobbsøknader, CV, aktivitetsplaner og lagrede søk
            </div>
            <a
              href="https://www.nav.no/no/Person/Arbeid/Arbeidsledig+og+jobbsoker"
            >
              Gå til din jobbsøk/CV
            </a>
            <div
              className="box-footer link-footer"
            >
              Kilde: 
              Arbeidsregisteret
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
